README.md

```diff
@@ -1,24 +1,80 @@
-**STILL IN DEVELOPMENT, NOT FIT FOR GENERAL CONSUMPTION, DO NOT USE IF NOT A DEVELOPER!**
-
 # Slic3r plugin for OctoPrint
 
-**WARNING** Not functional yet since slic3r's return codes are behaving unexpectedly. WIP for others to take a look at
```

Comment:
Have you tested under Windows before removing this? Because this is where I ran into a LOT of issues with Slic3r's behaviour on the command line, which is also why I marked the plugin as unsuitable for general consumption.

In a nutshell, slic3r kept returning negative return values even after slicing success, and varying values on top of that. Seemed to be completely random in behaviour.